### PR TITLE
Feature/payment

### PR DESCRIPTION
**Pull request by vannguyen141290**
I pair programmed with @Brendonjhawkins. We built the skeleton of payment page. Since BE is not built yet, we think we can leave the basic codes for now and maybe continue working on it next week if time allows.
One thing need to pay attention to is that if you review the changed files before PR#80 is merged, you will see many files and it's not necessary to be reviewed. please just check Cart.js and index.js file to see the changes.
Another thing is we added react-stripe-checkout in package.json in order to build payment component.

Loom[https://www.loom.com/share/0e1e7c12a50d4f3784dee0436e60d2bc](url)

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,21 +1,20 @@
...
+  <meta charset="utf-8" />
+  <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <meta name="theme-color" content="#000000" />
+  <meta name="description" content="Web site created using create-react-app" />
+  <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+  <!--
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
```
Comment (TraNequaFauntleroy) [inline public/index.html:14]: I love code comments!

File: public/index.html [hunk 2]
```diff
@@ -24,12 +23,13 @@
...
-    <div id="root"></div>
-    <!--
+  <title>Basic SPA</title>
+</head>
+
+<body>
+  <noscript>You need to enable JavaScript to run this app.</noscript>
+  <div id="root"></div>
+  <!--
       This HTML file is a template.
```
Comment (TraNequaFauntleroy) [inline public/index.html:33]: I dont know if this comment is necessary.

**Review by TraNequaFauntleroy — COMMENTED** [review]
Wow! You guys got a lot done! I know trying to get stripe to work was a little pain. Great job. Very clean code. I just question is all of the defaulted code comments are neccessary.

**Review by zach-morris-txt — APPROVED** [review]
It is a broad PR but I know you've has some branch history challenges that made it difficult to isolate specific features. The code is of excellent quality.

**Review by ashtilawat23 — APPROVED** [review]

**Pull request merged**